docs/development/security/gdpr_regulation.md

```diff
@@ -0,0 +1,156 @@
+
+The principles is summarized to the following list of tasks for the GIRAF project:
+* Determine what data is sufficient to fulfill the purpose of the purpose of the GIRAF system, and document both the details of the data, how and when it is collected and for which purpose.
+* Determine or define how the correctness of the data is ensured, and document it.
+* Define reasonable step to take, to ensure the integrity, confidentiality and security of the personal data, which applies both to organisational and technical levels.
+Determine for how long data has to be retained, for it to fulfill its purpose, along with both when and by what means, it should be deleted.
+
+### Legal Processing
+
+For any processing of personal data to be legal, there has to be appropriate lawful purpose to do so. There is a number of lawful purposes for processing personal data, one of which is the use of consent from the data subject, by means of a consent contract for example. The GDPR regulation defines several stipulations, regarding obtaining consent from the data subject, such as requiring high privacy and data protection by default, and any request for data collection to have a specific purpose and be of opt-in nature. Further, any processing of data regarding children, requires consent of the parent or guardian of the child. These requirements should be considered in the tasks regarding the GIRAF project.
+* Determine the lawful basis on which personal data is to be processed in the GIRAF system.
+    * Based on this, determine the most appropriate means of obtaining such lawful basis, such as seeking to establish a consent contract with the data subject.
+        * Should the data subject be a child, determine how the consent should be obtained from the parents or guardian.
+
+### Legal Rights of the Data Subject
+
+The GDPR regulation provides the data subject with several rights, which the data controller legally is required to inform the subject of. While the details of the exact rights are defined in the section describing the GDPR regulation, it is further necessary to determine when and how, the data subject should be informed. The tasks include:
+* Determine when and how the data subject is to be informed about their legal rights. The information should include:
+    * What is the extent of the data collection?
+    * Is any data transferred to third-parties and/or parties outside EU?
+    * Is any automated decision-making occurring, at what is the impact?
+    * What is the privacy rights of the data subject? (Details listed in the section about GDPR regulation).
```

Comment:
```suggestion
    * What is the privacy rights of the data subject? (Details listed in the section about GDPR regulation)
```

Reply:
Is it possible to crossref link to that section? Then I think it would be prudent to do.